### PR TITLE
Add roman-numerals exercise

### PR DESCRIPTION
**Pull request by ksyu**

### CODE REVIEW

File: exercises/roman-numerals/example.erl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+-define(DIGITS, [{"M", 1000}, {"CM", 900}, {"D", 500}, {"CD", 400},
+                 {"C", 100}, {"XC", 90},  {"L", 50}, {"XL", 40},
+                 {"X", 10}, {"IX", 9}, {"V", 5}, {"IV", 4}, {"I", 1}]).
+
+-spec numerals(non_neg_integer()) -> string().
+numerals(Number) ->
+    to_roman(Number, "", ?DIGITS).
+
+to_roman(0, Roman, _) -> Roman;
+to_roman(_, Roman, []) -> Roman;
```
Comment (NobbZ) [inline exercises/roman-numerals/example.erl]: Will this ever match? After we have processed `{"I", 1}` the first argument should be `0`, if not something went terribly wrong.

Comment (ksyu) [inline exercises/roman-numerals/example.erl]: After we have processed {"I", 1}, we got empty list of digits, so I think to_roman(_, Roman, []) will matched.

Comment (NobbZ) [inline exercises/roman-numerals/example.erl]: After we have processed `{"I", 1}` the accumulator (`N`) should be `0`, therefore when we have a match with an empty list, that means as well, that `N` is **not** `0`, which then means, that something went horribly wrong.

Comment (ksyu) [inline exercises/roman-numerals/example.erl]: I think see your point now, you are right, putting the empty list match line make no sense.

**Comment by NobbZ** [comment]
Thank you!

**Pull request merged**